### ISSUE TITLE
Canned Response Headers cannot contain spaces

### ISSUE THREAD

**Issue by alvarogarcia7** [quii/mockingjay-server#48]
Neither the key or the value of the headers in the canned response can contain spaces.

This blocks using cookies that contain expiry dates. 

The code responsible for this might be in https://github.com/quii/mockingjay-server/blob/master/mockingjay/validationHelpers.go

Example:

```
 - name: render Login form
   request:
     uri: /login
     regexuri: \/login.*
     method: GET
   response:
     code: 200
     body: 'non-empty body'
     Set-Cookie: 'Expires=Tue, 28 Apr 2037 13:46:30 GMT'
```

**Comment by alvarogarcia7** [comment]
Sent a pull request

**Comment by gypsydave5** [comment]
This got resolved a while ago by #49 ... closing

**Comment by alvarogarcia7** [comment]
Thanks @gypsydave5
